airbyte-integrations/connectors/source-amazon-seller-partner/source_amazon_seller_partner/streams.py

```diff
@@ -604,6 +604,15 @@ def parse_document(document):
         return reader
 
 
+class FlatFileOrdersReportsByLastUpdate(IncrementalReportsAmazonSPStream):
```

Comment:
Could you please explain how this stream is different from `FlatFileOrdersReports`, except that it's incremental, why does their schema differ?

Reply:
Because I've defined the schema according to a generated report.
  According to the [official doc*](https://sellercentral.amazon.com/help/hub/reference/201648780), the schema is not fully documented. Even more, I think it's not updated, e.g: a simple but important difference `order-id` seems to be renamed as `amazon-order-id`, but there is no mention in the doc.
  
  If you want, I can reuse the same as `FlatFileOrdersReports`, adapting the `amazon-order-id` and adding some other fields, because we don't have full visibility according to the API reference.

> *This list is not exhaustive. Additional fields may be available for the order report

Reply:
> If you want, I can reuse the same as FlatFileOrdersReports, adapting the amazon-order-id and adding some other fields, because we don't have full visibility according to the API reference.

This is was what I wanted to suggest, but as the report name is different between these two streams I think it's "normal" to expect different schemas, moreover, this API is very fragile, so I prefer the kind of iteration you made rather than the refacto we mention.